azure-pipelines.yml

```diff
@@ -460,22 +461,8 @@ stages:
             parallel: true
 
 - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - stage: publishBuildAssets
```

Comment:
No functionality change, just combining `publishBuildAssets` with the Build stage, which is how things used to be before https://github.com/dotnet/razor/pull/7037 (Arcade has since fixed the underlying problem).